src/components/PaperGridContainer.tsx

```diff
@@ -0,0 +1,22 @@
+import { Grid, makeStyles } from '@material-ui/core'
+import { ReactElement, ReactNode } from 'react'
+
+interface Props {
+  children: ReactNode | ReactNode[]
+}
+
+const useStyles = makeStyles(theme => ({
+  grid: {
+    backgroundColor: theme.palette.background.paper,
+  },
+}))
+
+export function PaperGridContainer({ children }: Props): ReactElement {
+  const classes = useStyles()
+
+  return (
+    <Grid className={classes.grid} container justifyContent="space-between" alignItems="center" direction="row">
+      {children}
+    </Grid>
```

Comment:
I feel like this particular element is mixing layout with design. I would think about if it makes sense to separate those two. Although I do see why you did it this way.